_episodes/03-technical.md

```diff
@@ -101,6 +101,38 @@ are modifying.
 > screenshared.
 {: .challenge}
 
+## Configuring and Sync Forks
+
+Forks are often used in open source development on GitHub 
+to make changes to a project without affecting the original repository. 
+You must first configure a remote that points to the upstream repository in Git.
```

Comment:
You must first configure a local remote that points to the original repository on GitHub.

- You can't add a remote in GitHub so is important to make clear that this is local only.
- I think we agreed to use "original" instead of "upstream". Use both term is confusing to novices.

Reply:
The problem is that Git will already refer to the fork as `origin` (which is very close to "original"), so I'm not sure that avoids confusion.

Reply:
I think `upstream` is a better nickname for the remote. Some text (or a figure) could be added to explain this. Does this document (https://github.com/dmgt/swc_github_flow/blob/master/for_novice_contributors.md#option-2---use-a-command-line-interface-with-git) have the information needed?